src/main.rs

```diff
@@ -109,7 +115,15 @@ fn main() {
 #[allow(unused_mut)] // TODO: Remove when fixed in chan_select
 fn run(matches: &ArgMatches) -> Result<()> {
     // Now we can start to run the i3bar protocol
-    print!("{{\"version\": 1, \"click_events\": true}}\n[");
+    let initialise = if matches.is_present("never-pause") {
+        format!(
+            "{{\"version\": 1, \"click_events\": true, \"stop_signal\": {}}}\n[",
+            nix::sys::signal::Signal::SIGUSR2 as i8
```

Comment:
```suggestion
            nix::sys::signal::Signal::SIGSTOP as i8
```
I thought this should be SIGSTOP

Reply:
Even with SIGSTOP it does not prevent the pause
I remember you asked me to test with signal 17 it seems to be associated with SIGCHLD

Reply:
@GChicha SIGSTOP is what i3 sends by default and is what causes i3status-rust to pause, and we should not ignore that signal if it is sent. However i3 lets us customising which signal it sends, so we can set it to something other than SIGSTOP to effectively prevent pausing from happening. I plan on using SIGUSR1 for reloading the config so using SIGUSR2 here for this one. I meant to use it in the original commit but accidently put 17 instead of 12.